components/sample-rhdevs-website/SignUpButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const SignUpButtonContainer = styled.button`
```

Comment:
Perhaps this component can be better renamed as a button as it seems to fulfil the functions of a button?

```suggestion
const SignUpButton = styled.button`
```

Reply:
hmm it conflicts with the function name if I name it as SignUpButton, is there any way around this?

Reply:
```suggestion
const StyledButton = styled.button`
```
Name it like this instead. Also, there's no need to make an _exclusive_ button just for signups. Instead make a generic button component that can be customised (i.e. take in params that set its display text, onClick function etc.)

Reply:
<img width="114" alt="image" src="https://user-images.githubusercontent.com/75428027/218317852-45be5968-ff81-4059-bf3f-a7ca469b5dfe.png">

mb for not mentioning earlier during the meeting, but i've placed some codes in the `sample-rhdevs-websites` subfolders, which contains codes that might be useful, so do refer to them if you're unsure of how to make for e.g. a button, or u can just copy them if they work!